sdk/translation/azure-ai-translation-text/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added support for AAD authentication.
```

Comment:
It seems like AAD support already exists and is not added in this PR. The change seems to related to adding new headers to requests. We should update the changelog accordingly.

Reply:
Updated the message.